### PR TITLE
Remnants

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

New vanilla Rival faction

Remnants - remnants of the AAF forces sponsored by CSAT, who want to regain control of the islands.

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
can use new [[], "SCRT_fnc_rivals_encounter_OccVsRivalsSkirmish"] call A3A_fnc_scheduler; command to spawn rivals vs occ skirmish
********************************************************

### CODE REVIEW

**Comment by wersal454** [comment]
big thanks to @UAC-MaxxLite for the custom flag

File: A3A/addons/core/CfgMarkers.hpp [hunk 1]
```diff
@@ -2,6 +2,13 @@ class CfgMarkers
 {
 	class flag_NATO;
 
+	class a3a_flag_aaf_torn_co: flag_NATO 
+	{
+		name = "AAF remnants";
```
Comment (Silence-112) [inline A3A/addons/core/CfgMarkers.hpp]:
```suggestion
		name = "AAF Remnants";
```

File: A3A/addons/core/Templates/Templates.hpp [hunk 2]
```diff
@@ -161,6 +161,16 @@ class Templates
         description = $STR_A3AP_setupFactionsTab_ll;
     };
 
+    class Vanilla_Remnants : Vanilla_Base
+    {
+        side = "Riv";
+        flagTexture = QPATHTOFOLDER(Templates\Templates\Vanilla\flag_aaf_torn_co.paa);
+        name = "A3 Remnants";
```
Comment (Silence-112) [inline A3A/addons/core/Templates/Templates.hpp]:
```suggestion
        name = "A3 AAF Remnants";
```

File: A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Riv_Exegermenos.sqf [hunk 3]
```diff
@@ -585,11 +564,11 @@ if (_hasRF) then {
...
 
 if (_hasGM) then {
-    (_loadoutData get "facewear") append [
+    (_loadoutData get "fullmask") append [
         "gm_ge_facewear_stormhood_blk",
         "gm_ge_facewear_stormhood_dustglasses_blk",
         "gm_ge_facewear_stormhood_brd"
-        ];
+    ];
 } ;
```
Comment (Silence-112) [inline A3A/addons/core/Templates/Templates/Vanilla/Vanilla_Riv_Exegermenos.sqf]:
```suggestion
};
```

File: A3A/addons/scrt/Stringtable.xml [hunk 4]
```diff
@@ -387,6 +387,13 @@
                 <Korean>호라이즌 제도 방위군(HIDF).\n호라이즌 제도 공화국의 군대이며, 많은 여분의 NATO 장비를 사용합니다.</Korean>
                 <French>Force de défense des îles Horizon (HIDF).\nLes forces armées de la République des îles Horizon utilisent de nombreux équipements excédentaires de l'OTAN.</French>
             </Key>
+            <Key ID="STR_A3AP_setupFactionsTab_remnants">
+                <Original>Remnants.\nRemnants of the AAF forces sponsored by CSAT, who want to regain control of the islands..</Original>
```
Comment (Silence-112) [inline A3A/addons/scrt/Stringtable.xml:391]:
```suggestion
                <Original>Remnants.\nRemnants of the AAF forces sponsored by CSAT, who want to regain control of the islands.</Original>
```

File: A3A/addons/scrt/Stringtable.xml [hunk 5]
```diff
@@ -387,6 +387,13 @@
...
+                <Russian>Remnants.\nRemnants - остатки сил AAF спонсируемые CSAT, желающие вернуть контроль над островами.</Russian>
+                <Chinesesimp>Remnants.\nPLACEHOLDER。</Chinesesimp>
+                <Korean>엑세게르메노스.\nPLACEHOLDER.</Korean>
+                <French>Remnants.\nPLACEHOLDER.</French>
```
Comment (Silence-112) [inline A3A/addons/scrt/Stringtable.xml]: Might aswell just remove these translations so it uses english, unless somebody gives a translation

**Review by Silence-112 — CHANGES_REQUESTED** [review]

File: A3A/addons/scrt/Stringtable.xml (hunk 5 above)
Comment (Silence-112) [reply A3A/addons/scrt/Stringtable.xml]: Ignore the russian one, didn't realise I selected it

**Review by Silence-112 — APPROVED** [review]

**Pull request merged**